### DISCUSSION TITLE
jwt example with webflux

### DISCUSSION THREAD

**Discussion by Horoshiy** [Netflix/dgs-framework#1064] [Q&A]
Please, send example of JWT authorization via dgs and webflux, Every time I have HttpBasic window on /graphiql/ page. Methods with Secured annotation not working, Thanks!

**Comment by srinivasankavitha** [comment]
The set up for this shouldn't be any different from doing the same with the webmvc stack, afaik. Have you tried setting breakpoints and debugging to check if the appropriate roles are coming through with the token to match what is in @Secured?

**Comment by defro** [comment]
Hello @Horoshiy 
Did you find a example ? If yes can you share please ?
